src/main/java/com/capitalone/dashboard/webhook/github/GitHubPullRequestV3.java

```diff
@@ -519,4 +523,14 @@ protected void updateMatchingCommitsInDb(Commit commit, GitRequest pull) {
     private long getTimeStampMills(String dateTime) {
         return StringUtils.isEmpty(dateTime) ? 0 : new DateTime(dateTime).getMillis();
     }
+
+    private boolean isValidEvent(String action) {
+        return Arrays.asList(PullRequestEvent.Opened,
+                PullRequestEvent.Edited,
+                PullRequestEvent.Closed,
+                PullRequestEvent.Reopened,
+                PullRequestEvent.Merged,
+                PullRequestEvent.Synchronize).contains(PullRequestEvent.fromString(action));
+    }
+
```

Comment:
may be define a static final at the class level?

Reply:
@tabladrum  PullRequestEvent is an enum https://github.com/Hygieia/hygieia-core/pull/156